config/environment.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return {};
+  var ENV = {
+    contentSecurityPolicy: {
+      'default-src': "'none'",
+      'style-src': "'self' 'unsafe-inline' 'unsafe-eval' http://feather.aviary.com/3.1.0.284/css/feather_theme_aviary_dark.css http://feather.aviary.com/3.1.0.284/css/feather_theme_aviary_light.css",
```

Comment:
just wondering if this is still needed since the aviary-editor lib is baked into the addon